modules/ai/assets/js/editor/ai-behavior.js

```diff
@@ -28,6 +30,37 @@ export default class AiBehavior extends Marionette.Behavior {
 		};
 	}
 
+	getTextualContent() {
+		const pageTextContent = [];
+
+		// Use elementorFrontend to loop through all the elements in the Elementor editor
+		const clonedCurrentLayout = window.elementor.$previewContents[ 0 ]?.cloneNode( true );
+		clonedCurrentLayout.querySelectorAll( '.elementor-editor-element-settings, #elementor-add-new-section, .elementor-add-section-inner, header, footer' )
```

Comment:
The content may include header and footer tags

Reply:
I added header and footer to the query, why wouldn't it be removed?